src/components/Switch.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import defaultTheme from '../themes/DefaultTheme';
+import darkTheme from '../themes/DarkTheme';
+
+const Switch = ({
+  label,
+  position,
```

Comment:
Imho, we need to change `position` to `labelPosition` as we don't define the position of the switch itself in the app, but position of the label relative to the switch button.